src/cmake/install_tbb.cmake

```diff
@@ -206,10 +206,12 @@ if(THREADING MATCHES "^(TBB|TBB_AUTO)$" AND
                                DEPENDS tbb)
         list(APPEND core_dev_components tbb_dev)
 
-        install(FILES "${TBBROOT}/cmake/TBBConfig.cmake"
-                      "${TBBROOT}/cmake/TBBConfigVersion.cmake"
-                DESTINATION "${IE_TBB_DIR_INSTALL}/cmake"
-                COMPONENT tbb_dev)
+        if(EXISTS "${TBBROOT}/cmake")
```

Comment:
why does it exists?
If `cmake` folder is a sub-folder of `lib`, we need to make sure they are installed to different cmake components:
- tbb_dev for development files like cmake, headers
- tbb for libraries only